packages/shared/src/components/CreateMyFeedButton.tsx

```diff
@@ -24,20 +20,16 @@ const getAnalyticsEvent = (
 });
 
 export default function CreateMyFeedButton({
-  flags,
   action,
 }: CreateMyFeedButtonProps): ReactElement {
   const { trackEvent } = useContext(AnalyticsContext);
-  const buttonCopy = getFeatureValue(Features.MyFeedButtonCopy, flags);
-  const buttonColor = getThemeColor(
-    getFeatureValue(Features.MyFeedButtonColor, flags),
-    Features.MyFeedButtonColor.defaultValue,
-  );
-  const explainerCopy = getFeatureValue(Features.MyFeedExplainerCopy, flags);
-  const explainerColor = getThemeColor(
-    getFeatureValue(Features.MyFeedExplainerColor, flags),
-    Features.MyFeedExplainerColor.defaultValue,
-  );
+  const buttonCopy = 'Choose tags';
+  const cabbageColors = {
+    border: 'border-theme-color-cabbage',
+    shadow: 'shadow-2-cabbage',
+    button: 'btn-primary-cabbage',
+  };
```

Comment:
I think we can now make this in-line as it is not used anywhere else.

Reply:
It is technically inline now, but two elements use it, so decided to make it a reusable one.

Reply:
From the PR, each of the properties is only used once. Can you point out where either of the properties were used again?

Reply:
Sorry you where right 🙇